docs/topics/about-home-var.adoc

```diff
@@ -10,7 +10,10 @@ This guide uses the `<MTA_HOME>` replaceable variable to denote the path to your
 
 [NOTE]
 ====
-For Windows operating systems, you must extract the {ProductShortName} `.zip` file to a folder named `mta` to avoid a `Path too long` error.
+If you are installing on a Windows operating system:
+
+. Extract the `.zip` file to a folder named `mta` to avoid a `Path too long` error _or_ extract the file with link:https://www.7-zip.org/download.html[7-Zip] to a folder of any name you choose.
+. If a *Confirm file replace* window is displayed during extraction, click *Yes to all*.
 ====
 
```

Comment:
@RichardHoch 
Is 'or' in italics to emphasize it? I would suggest breaking the sentence into two instead. "...a `Path too long` error. Alternatively, you can extract the file..."

Reply:
Done.